embedded/Localization/fr-FR.json

```diff
@@ -12,31 +12,31 @@
-  "Options.OPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.Setting.NextSongRadioBinding]": "Next radio song key",
-  "Options.OPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod/NextSongRadioBinding/binding]": "Next radio song binding",
-  "Options.OPTION_DESCRIPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.Setting.NextSongRadioBinding]": "Keyboard shortcut to skip the current song.",
-  "Options.OPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.Setting.PrevSongRadioBinding]": "Previous radio song key",
-  "Options.OPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod/PrevSongRadioBinding/binding]": "Prev radio song binding",
-  "Options.OPTION_DESCRIPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.Setting.PrevSongRadioBinding]": "Keyboard shortcut to go back to the latest song.",
-  "Options.OPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.Setting.VolumeDownRadioBinding]": "Radio volume up key",
-  "Options.OPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod/VolumeDownRadioBinding/binding]": "Radio volume up binding",
-  "Options.OPTION_DESCRIPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.Setting.VolumeDownRadioBinding]": "Keyboard shortcut to increase the volume of the radio.",
-  "Options.OPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.Setting.VolumeUpRadioBinding]": "Radio volume down key",
-  "Options.OPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod/VolumeUpRadioBinding/binding]": "Radio volume down binding",
-  "Options.OPTION_DESCRIPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.Setting.VolumeUpRadioBinding]": "Keyboard shortcut to decrease the volume of the radio."
+  "Options.TAB[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.MixNetwork]": "Chaîne de mix",
+  "Options.OPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.Setting.MixNetworkEnabled]": "Activer la chaîne de mixes",
+  "Options.OPTION_DESCRIPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.Setting.MixNetworkEnabled]": "Activer la chaîne de mixes. N'oubliez pas de recharger la radio.",
+  "Options.OPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.Setting.MixNetworkClearQueue]": "Vider la file d'attente des clips",
+  "Options.OPTION_DESCRIPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.Setting.MixNetworkClearQueue]": "Vider la file d'attente des pistes pour en faire une nouvelle à partir de votre sélection.",
+  "Options.OPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.Setting.MixNetworkFinishCurrentClip]": "Terminer le clip",
+  "Options.OPTION_DESCRIPTION[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.Setting.MixNetworkFinishCurrentClip]": "Terminez le clip actuel pour jouer instantanément la nouvelle file d'attente.",
+  "Options.TAB[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.Keybinds]": "Raccourcis clavier",
+  "Options.GROUP[ExtendedRadio.ExtendedRadio.ExtendedRadioMod.RadioControls]": "Raccourcis clavier de contrôle de la radio",
```

Comment:
I'd just say `Commandes de radio`, which aligns more with the original. This translates to "keyboard shortcuts to control the radio"